app/views/incidents/_form.html.haml

```diff
@@ -19,8 +19,10 @@
       .field
         = form.label :bus
         = form.text_field :bus, id: :incident_bus
-      .field
+      .field.high-contrast-options
```

Comment:
That's a good point. Truth be told, these are the only select tags on the site, but might as well have all future select tags pass the test too. a630403